modules/smart-agent_zookeeper/conf/01-zookeeper-latency.yaml

```diff
@@ -0,0 +1,22 @@
+module: zookeeper
+name: zookeeper-latency
+transformation: false
+aggregation: true
+exclude_not_running_vm: true
+disabled: false
+signals:
+  signal:
+    metric: "gauge.zk_avg_latency"
+rules:
+  critical:
```

Comment:
I'm not sure that a high latency on one server should trigger a critical alter. 
Maybe 2 detectors:
* one that trigger major and critical if all servers in a cluster have high latency
* one that trigger major for a single server high latency